73063.typ

```diff
@@ -362,6 +370,7 @@
 		checklist_group("Left"),
 		([Main wheel tire], [CHECK INFLATION]),
 	)
+	#colbreak()
```

Comment:
Removing the "Tach time ... RECORD" entry above cauled the "Preflight (continued)" box to fit in the left column, which broke the formatting and pushed the light gun signal box to the next page. This column break restores the formatting.